Tests/oss-fuzz/test_fuzzers.py

```diff
@@ -48,6 +48,7 @@ def test_fuzz_images(path):
         fuzzers.disable_decompressionbomb_error()
 
 
+@pytest.mark.skipif(not features.check("freetype2"), reason="freetype2 not available")
```

Comment:
Can we also use this one here? 

```suggestion
@skip_unless_feature("freetype2")
```

Reply:
This is in the oss-fuzz subdirectory, rather than just in Tests. I might be missing something, but all my attempts to `skip_unless_feature` via `helper` from here lead to `ImportError: attempted relative import with no known parent package`

Reply:
<kbd>option</kbd>+<kbd>enter</kbd> in PyCharm suggests `from Tests.helper import skip_unless_feature`, which works for me locally:

![image](https://github.com/python-pillow/Pillow/assets/1324225/db4433be-16b5-4a39-a1de-394efc04637f)

Reply:
Thanks, that works. I've updated the commit.